### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by Crocodile73** [valinet/ExplorerPatcher#285] [General]
![unknown-2](https://user-images.githubusercontent.com/16454525/140553930-e3e22efe-a894-467b-8c59-f9b9501e44ee.png)

Any plans for the win 7 option. Third party themes don't look good unless you use this as in startallback.

**Comment by valinet** [comment]
No plans. OldNewExplorer works very well.

**Reply by Crocodile73** [reply]
oldnewexplorer doesn't work for Windows 11

**Reply by valinet** [reply]
It does. I use it as we speak.

**Reply by Crocodile73** [reply]
Using the winaero tweaker way? Otherwise it doesn't work for Windows 11.

**Reply by Crocodile73** [reply]
Ah ok I see what you mean.
